theseus_gui/src/components/ui/Instance.vue

```diff
@@ -124,4 +150,16 @@ const props = defineProps({
     color: #000;
   }
 }
+
+.instance-project-item {
+  width: 100%;
+  height: auto;
+  margin: 2rem auto;
```

Comment:
This feels like way too much margin. 0.75rem is what the main site uses between each card, I would reco that